src/components/back-link-helper/back-link-helper.js

```diff
@@ -0,0 +1,30 @@
+function BackLinkHelper($module, window, document) {
+  this.$module = $module;
+  this.window = window;
+  this.document = document;
+}
+
+BackLinkHelper.prototype.init = function init() {
+  // do nothing if History API is absent
+  if (this.window.history) {
+    // store referrer value to cater for IE
+    const docReferrer = this.document.referrer;
+
+    // prevent resubmit warning
+    if (this.window.history.replaceState && typeof this.window.history.replaceState === 'function') {
+      this.window.history.replaceState(null, null, this.window.location.href);
+    }
```

Comment:
I wonder if we'd want this as a separate thing, I could imagine wanting this on all pages, even where back links are not mimicking the browser back button via javascript

Reply:
(future work)